app/views/admin/gdpr_delete_requests/index.html.erb

```diff
@@ -1,8 +1,8 @@
 <%= javascript_packs_with_chunks_tag "admin/users/gdprDeleteRequests", defer: true %>
 <div id="gdpr-delete-requests-content" class="crayons-card overflow-admin-main-layout-padding p-4 m:p-0">
   <header class="flex flex-col l:flex-row justify-content-between l:items-center p-0 m:p-7 pb-4">
-    <h1 class="crayons-title ml-3 l:ml-0">Members<span class="screen-reader-only"> (GDPR Delete Requests)</span></h1>
-    <%= form_with url: admin_gdpr_delete_requests_path, method: :get, local: true, class: "grow-1 mb-3 m:mb-0" do |f| %>
+   <h1 class="crayons-title ml-3 l:ml-0">Members<span class="screen-reader-only"> (GDPR Delete Requests)</span></h1>
```

Comment:
We have changed the text for to `Invited Members` earlier, so similar to that should this text be changed to `GDPR Actions` ?

<img width="462" alt="Screenshot 2023-01-13 at 3 58 05 PM" src="https://user-images.githubusercontent.com/9396084/212299259-135561a8-05de-4204-9419-56c2c74dd416.png">

Reply:
Makes sense to em for them to be the same, I'll update this.

Reply:
Looking at this again, perhaps its not so intuitive to make this change. The table is still a table of Members and not only GDPR actions so perhaps it should read Members (GDPR Actions) or maybe I should not have updated `Members` to `Invited Members`  on the previous page. I was trying to align the nav bar with the heading of the page but since @anujbhavsar96 had put a lot of thought and effort into the Member Manager, I'd like to get his thoughts on this as well?

Reply:
The table's title should read the same as on the left sidebar. Didn't we have this that way only?

Let's say for Invited Members page title should be `Invited Members,` and for GDPR Actions it should be `GDPR Actions`

Keeping the same title for all three tables looks misleading.

Reply:
Sounds good.